### DISCUSSION TITLE
Hiding Bard sets from output has no effect

### DISCUSSION THREAD

**Discussion by brighttank** [statamic/cms#4887] [Troubleshooting]
When I add Sets to a Bard field there is a toggle option to toggle between "Included in output" and "Hidden in output". However if I'm rendering using a template that renders each block based on the set type then this option has no effect.
```
{{ my_bard_field }}
    {{ if type == "text" }}
      ... the text part
    {{ elseif type == "images" }}
      ... show images
    {{ /if }}
{{ /my_bard_field }}
```
In order to get the button to have effect I have to manually check for the attribute. This is further complicated by the fact that if the set is enabled then the attribute is removed. So I have to include an explicit test for falseness, eg.

`{{ elseif type == "images" and enabled !== false }}`

Is this a bug or am I doing something wrong?

I'm on version 3.2.24

**Comment by espenlg** [comment]
I'm on 3.2.25 and it's working fine here. Both in Live preview and on the site. I've done it exactly like you 🤔

**Comment by edalzell** [comment]
Are you using any caching? Have you cleared the cache?
